docs/runbooks/NoLeadingVirtOperator.md

```diff
@@ -1,42 +1,73 @@
+<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
+
 # NoLeadingVirtOperator 
 
 ## Meaning
 
-This alert is triggered when the system detects that in the past 10 minutes, there is no virt-operator pod holding the leader lease despite there are one or more virt-operator pod is in `Ready` state. The alert suggests that there is no operating virt-operator pod. 
+This alert fires when the system detects that in the past 10 minutes, no `virt-operator` pod has been holding the leader lease, although one or more virt-operator pods are in the `Ready` state. The alert suggests that there is no operating `virt-operator` pod. 
```

Comment:
The first sentence is a bit hard to read. I think it would be more readable if you break it into two sentences or perhaps shorten it. There are times when a passive voice is useful!

Maybe this: "This alert fires when no `virt-operator` pod with a leader lease has been detected for 10 minutes, although the `virt-operator` pods are in a `Ready` state. The alert indicates that there is no leader pod."

"operating pod" is a little weird. The normal term for a pod with a leader lease is "leader pod". One pod is a leader and the rest are there for redundancy.